### ISSUE TITLE
Can't able to do Default Zoom 2 when scanner is launched

### ISSUE THREAD

**Issue by HarshDadhania** [yudielcurbelo/react-qr-scanner#89]
I want to set Default Zoom 2 when scanner is launched, currently I'm not able to find property for that.

**Comment by MananJa13ba** [comment]
For anyone using typescript:
const zoomObj: any = { zoom: 3 }
constraints={{ advanced: [zoomObj] }}

**Comment by dangdinhvan** [comment]
> For anyone using typescript: const zoomObj: any = { zoom: 3 } constraints={{ advanced: [zoomObj] }}

it's not working for me. constraints={{ advanced: MediaTrackConstraintSet[] }} not include zoom. So any other solution?

**Comment by MananJa13ba** [comment]
> > For anyone using typescript: const zoomObj: any = { zoom: 3 } constraints={{ advanced: [zoomObj] }}
> 
> it's not working for me. constraints={{ advanced: MediaTrackConstraintSet[] }} not include zoom. So any other solution?

Yes, MediaTrackConstraintSet lacks a zoom property, which is why we initialized it with type 'any.' If you're using the scanner component in a mobile application, the camera should reflect the zoom level you selected.
